salt/metalk8s/defaults.yaml

```diff
@@ -51,4 +53,36 @@ kubelet:
     authorization:
       mode: AlwaysAllow
 
+ca:
+  cert:
+    name: /etc/kubernetes/pki/ca.crt
+    signing_private_key: /etc/kubernetes/pki/ca.key
+    CN: kubenetes
+    keyusage: "critical digitalSignature, keyEncipherment, keyCertSign"
+    basicConstraints: "critical CA:true"
+    days_valid: 3650
+  signing_policies:
+    api:
+      - minions: '*'
```

Comment:
Can we limit this? Cfr. comment in PR overview.

Reply:
Yes but we need to know which minions to put here. And for this we can only do basic globbing on minion-id for me it's hard to automatically put the right thing here

Reply:
Can you create a tracking ticket?

Reply:
Done